sdcm/cluster.py

```diff
@@ -2448,8 +2448,8 @@ def prepare_files_for_archive(self, fileslist):
                 self.remoter.run('cp -r %s %s' % (f, new_full_path), ignore_status=True)
         return root_dir
 
-    def generate_coredump_file(self, restart_scylla=True):
-        self.log.info('Generate scylla core')
+    def generate_coredump_file(self, restart_scylla=True, node: Any = None):
+        GeneratingCoreDumpEvent(node=node).publish()
```

Comment:
What's wrong with using InformationalEvent

Reply:
Roy asked for an error event. and it makes sense since some error must have happened to the node in this scenario,

Reply:
first, unittest are failing with this change:
```
ValueError: There is no max severity configured for GeneratingCoreDumpEvent
```

What's wrong with:

```
InformationalEvent(severity=Severity.ERROR)
```

adding an event to sct would normally mean there more then one place it's going to called from
and there more specific information it's going to hold.

in this case you just want an error event raised, with a clear message, I think `InformationalEvent` is enough

Reply:
InformationalEvent(severity=Severity.ERROR) is good enough.
We don't need a new type of event.

Reply:
according to @fruch , he meant using `InfoEvent` (not InformationalEvent). but then calling `InfoEvent(severity=Severity.ERROR)` may not be sufficient in order to override severity.

Reply:
fixed